app/controllers/quick_reservations_controller.rb

```diff
@@ -0,0 +1,109 @@
+# frozen_string_literal: true
+
+class QuickReservationsController < ApplicationController
+  layout "quick_reservation"
```

Comment:
What's the goal of the new layout here?

Reply:
The mockup lacks the normal header, footer, and side panel. So it seemed like another layout would be needed